composer.json

```diff
@@ -61,7 +61,7 @@
         "oat-sa/lib-lti1p3-ags": "^1.2",
         "oat-sa/lib-lti1p3-core": "^6.0.0",
         "oat-sa/generis" : ">=15.22",
-        "oat-sa/tao-core" : ">=50.24.6"
+        "oat-sa/tao-core" : "dev-feature/MS-2756/add-userpilot-to-tao-3.x"
```

Comment:
Do not forget to replace by the proper release before merge

Reply:
Yes, I am aware of it. First I need approval for depending PR, merge it, and then update the new version here.

Reply:
Updated to >=54.8.0